docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst

```diff
@@ -3,16 +3,125 @@
+    - Determined a new storage location where you would like to store Expectations. This can either be a local path, or a path to a network filesystem.
+
+Steps
+-----
+
+1. First create a new folder where you would to store your Expectations, and move your existing Expectation files over to the new location. In our case, the name of the Expectations file is ``npi_expectations`` and the path to our new storage location is ``/shared_expectations``.
+
+.. code-block:: bash
+
+    # in the great_expectations folder
+    mkdir shared_expectations
+    mv expectations/npi_expectations.json shared_expectations/
+
+
+2. Next open the ``great_expectations.yml`` file and look for the following lines.
+
+.. code-block:: yaml
+
+    expectations_store_name: expectations_store
+
+    stores:
+        expectations_store:
+            class_name: ExpectationsStore
+            store_backend:
+                class_name: TupleFilesystemStoreBackend
+                base_directory: expectations/
+
+
+The configuration file tells Great Expectations to look for Expectations in a Store called ``expectations_store``. The ``base_directory`` for ``expectations_store`` is set to ``expectations/`` by default.
+
+3. Update your configuration following the example below. This example would change the Store name to ``shared_expectations_filesystem_store`` with the ``base_directory`` set to ``shared_expectations/``.
+
+    Paths are relative to the directory where ``great_expectations.yml`` is stored.
+
+
+.. code-block:: yaml
+
+    expectations_store_name: shared_expectations_filesystem_store
+
+    stores:
+        shared_expectations_filesystem_store:
+            class_name: ExpectationsStore
+            store_backend:
+                class_name: TupleFilesystemStoreBackend
+                base_directory: shared_expectations/
+
+
+
+4. Confirm that the location has been updated by running ``great_expectations store list``.
+
+.. code-block:: bash
+
+    great_expectations store list
+
+    3 Stores found:
+
+    - name: shared_expectations_filesystem_store
+    class_name: ExpectationsStore
+    store_backend:
+        class_name: TupleFilesystemStoreBackend
+        base_directory: shared_expectations/
+
+
+5. Confirm that Expectations can be read from the new storage location by running ``great_expectations suite list``.
+
+.. code-block:: bash
+
+    great_expectations suite list
+
+    1 Expectation Suite found:
+        - npi_expectations
+
+
+Additional Notes
+----------------
+
+- For best practices, we highly recommend that you store Expectations in a version-control system like Git. The JSON format of Expectations will allow for informative diff-statements and effective tracking of modifications. In the example below, 2 changes have been made to ``npi_expectations``.  The Expectation ```expect_table_column_count_to_equal`` was been changed from ``330`` to ``333`` to ``331``.
```

Comment:
This is the blurb that recommends storing Expectations on Git. Does it need a `punchy` conclusion?